### ISSUE TITLE
I found that the task will not startup on time if the previous task didn't finished.

### ISSUE THREAD

**Issue by xiaoshao** [merencia/node-cron#23]
I create a con job as follows.

`
var cron = require("node-cron");
var sleep = require('sleep');
var test = 1;

function start() {
    cron.schedule("* * * * * *", function () {
        console.log("started");
        sleep.sleep(10);
        test = test + 1;
        console.log(test);
        console.log("finished...");
    });
}

module.exports = start();
`

The output is 

`
   node src/start
started
2
finished...
started
3
finished...
started
4
finished...
started
5
finished...
started
6
finished...
started
7
finished...
started
^C
`

The cron job should startup per seconds on time no matter the previous job is finished or not, right?

**Comment by merencia** [comment]
Hello @xiaoshao 

Internally node-cron uses a `setInterval` verify if a task should be performed, so when the execution is greater than the delay, the next verification will be queued.

http://javascript.info/tutorial/settimeout-setinterval#the-real-delay-of-setinterval  

About the behavior, I think if your job execution time is greater than 1 sec and your scheduling is every second, the next execution will be performed when the current execution has finished.
In other words, in one minute your task probably will be performed less then 60 times.

**Comment by xiaoshao** [comment]
there will be a lot tasks in the queue after a long time.

**Comment by merencia** [comment]
I think so
